marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/writer/HubDataWriterFactory.java

```diff
@@ -30,45 +29,37 @@
 public class HubDataWriterFactory extends LoggingObject implements DataWriterFactory<InternalRow> {
 
     private StructType schema;
-    private HubClient hubClient;
```

Comment:
@anu3990  @SameeraPriyathamTadikonda  I had to get rid of this field - Spark complained about this field because it's not serializable.